<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStu</class>
 <widget class="QDialog" name="addStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Agency FB&quot;;
color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>add student information</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>name：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEd_name">
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>Please enter your name</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ID：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEd_id">
       <property name="placeholderText">
        <string>Please enter the 6-digit student ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,3,4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>sex：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_male">
       <property name="text">
        <string>male</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sex_buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_female">
       <property name="text">
        <string>female</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">sex_buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,8,0">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>age：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="age_comboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,8,0">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>department：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dep_comboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>School of Computer Science &amp; Technology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>School of Business</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>333</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>School of Metallurgical Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>School of Electrical &amp; Information Engineering</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>interest：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>football</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ins_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>skf</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ins_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>sing</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ins_buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>dance</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ins_buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,1">
     <item>
      <widget class="QPushButton" name="btn_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancle">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ins_buttonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="sex_buttonGroup"/>
 </buttongroups>
</ui>
